common/logging.go

```diff
@@ -82,56 +86,143 @@ func (l ConsoleLogger) IsLogLevel(level LogLevel) bool {
 	return l.LogLevel >= level
 }
 
+// Error logs 'Error' message.
 func (l ConsoleLogger) Error(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelError {
 		prefix := "[ERROR] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Warning logs 'warning' message.
 func (l ConsoleLogger) Warning(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelWarning {
 		prefix := "[WARNING] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Notice logs 'Notice' message.
 func (l ConsoleLogger) Notice(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelNotice {
 		prefix := "[NOTICE] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Info logs 'Info' message.
 func (l ConsoleLogger) Info(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelInfo {
 		prefix := "[INFO] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Debug logs 'Debug' message.
 func (l ConsoleLogger) Debug(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelDebug {
 		prefix := "[DEBUG] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// Trace logs 'Trace' message.
 func (l ConsoleLogger) Trace(format string, args ...interface{}) {
 	if l.LogLevel >= LogLevelTrace {
 		prefix := "[TRACE] "
 		l.output(os.Stdout, prefix, format, args...)
 	}
 }
 
+// output writes `format`, `args` log message prefixed by the source file name, line and `prefix`
+//noinspection GoUnhandledErrorResult
```

Comment:
Remove the "noinspection" comment, seems to be for some linter that we are not using.